samples/ControlCatalog/Pages/ProgressBarPage.xaml

```diff
@@ -6,15 +6,19 @@
     <TextBlock Classes="h2">A progress bar control</TextBlock>
 
     <StackPanel>
+      <CheckBox
+          x:Name="cbShowProgress"
+          Margin="10,16,0,0"
+          Content="Show Progress Text" />
       <StackPanel Orientation="Horizontal"
                   Margin="0,16,0,0"
                   HorizontalAlignment="Center"
                   Spacing="16">
         <StackPanel Spacing="16">
-          <ProgressBar Value="{Binding #hprogress.Value}" />
+          <ProgressBar ShowProgressText="{Binding ElementName=cbShowProgress, Path=IsChecked}" Value="{Binding #hprogress.Value}" />
```

Comment:
I think we can use the short-form binding syntax here like with the `Value` property: `ShowProgressText="{Binding #cbShowProgress.IsChecked}"`: http://avaloniaui.net/docs/binding/binding-to-controls